libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboardStyles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
the styles file names need to be .styles.ts, this one is missing the . as in:
VisionExplanationDashboardStyles.ts -> VisionExplanationDashboard.styles.ts
see other files for example